### PR TITLE
Pn/feature dashboard

### PR DESCRIPTION
**Pull request by nemoipaha**

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -12,11 +12,30 @@ class UserController extends ApiController
...
     }
+
+    public function getUsersByStatus($users, $status) {
+        $filteredUsers = [];
+
+        foreach ($users as $user) {
+            if ($user['status'] == $status) {
+                array_push($filteredUsers, $user);
+            }
+        }
```
Comment (binaryvo) [inline app/Http/Controllers/UserController.php]: FYI: PHP has awesome function for this action: array_filter(); So you can use it next time.

File: resources/assets/js/app/views/dashboard/DashboardLayout.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+var Marionette = require('backbone.marionette');
+var logger = require('../../instances/logger');
+var _ = require('underscore');
+var CountersCollectionView = require('./CountersCollection');
+var DashBoardProfileView = require('./DashBoardProfile');
+var DashBoardMainView = require('./DashBoardMain');
```
Comment (binaryvo) [inline resources/assets/js/app/views/dashboard/DashboardLayout.js]: var DashBoardMainView = require('./DashBoardMain');

DashBoardMain - please check filename and file that you requireing

**Comment by binaryvo** [comment]
Please fix all remarks and re-open PR

**Comment by nemoipaha** [comment]
remarks fixed

**Pull request merged**